cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
The 'validate' method is a very important part of the serializer. It would be helpful to add a comment explaining what this method does, what it returns, and why it's necessary.